Tsx parser imports hoisted lodash version

[This line](https://github.com/facebook/jscodeshift/blob/5910a4dcde431813c2e34a3a6f47e8800d275eb6/parser/tsx.js#L11) can cause transforms that have a second transitive dep on lodash to crash unexpectedly when lodash is unhoisted. I think this could be addressed a couple ways- either jscodeshift could express a dep on lodash or the tsx parser could remove the reference to lodash. I lean towards removing the lodash usage entirely (or at least not importing all of lodash to call one util one time).

I think I'm consistently hitting this.

```
Transformation error (Cannot find module 'lodash' Require stack: - /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/parser/tsx.js - /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/src/getParser.js - /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/src/Worker.js)
Error: Cannot find module 'lodash'
Require stack:
- /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/parser/tsx.js
- /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/src/getParser.js
- ```

--- a/package.json
+++ b/package.json
@@ -58,6 +58,7 @@
     "roots": [
       "src",
       "bin",
+      "parser",
       "sample"
     ]
   }

--- a/parser/__tests__/.eslintrc
+++ b/parser/__tests__/.eslintrc
@@ -1,0 +1,8 @@
+{
+  "globals": {
+    "jest": true
+  },
+  "env": {
+    "jasmine": true
+  }
+}

--- a/parser/__tests__/tsx-test.js
+++ b/parser/__tests__/tsx-test.js
@@ -1,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/*global jest, describe, it, expect*/
+
+'use strict';
+
+jest.mock('@babel/parser')
+const babylon = require('@babel/parser');
+
+const tsxParser = require('../tsx');
+
+describe('tsxParser', function() {
+  describe('parse', function() {
+    it('extends the ts config with jsx support', function() {
+      const parser = tsxParser();
+      parser.parse('"mock content";');
+
+      expect(babylon.parse).toHaveBeenCalledTimes(1);
+      expect(babylon.parse.mock.calls[0]).toMatchSnapshot();
+    });
+  });
+});

--- a/parser/tsx.js
+++ b/parser/tsx.js
@@ -8,11 +8,11 @@
 
 'use strict';
 
-const _ = require('lodash');
 const babylon = require('@babel/parser');
 const baseOptions = require('./tsOptions');
 
-const options = _.merge(baseOptions, { plugins: ['jsx'] });
+const options = Object.assign({}, baseOptions);
+options.plugins = ['jsx'].concat(baseOptions.plugins);
 
 /**
  * Doesn't accept custom options because babylon should be used directly in

